cpp/ble/arbitrary-data-transmission-service/Central/Program.cs

```diff
@@ -121,18 +124,123 @@
 }
 await PrintCharacteristicInfo(transmissionCharacteristic);
 
+// Get the receive characteristic
+Console.WriteLine($"Searching for Receive characteristic.");
```

Comment:
It would be great if we could create a class for the ADT service on the c# side.